src/coreclr/tools/Common/Internal/Runtime/ModuleHeaders.cs

```diff
@@ -67,6 +67,7 @@ public enum ReadyToRunSectionType
         OwnerCompositeExecutable = 116, // Added in 4.1
         PgoInstrumentationData = 117, // Added in 5.2
         ManifestAssemblyMvids = 118, // Added in 5.3
+        CrossModuleInlineInfo = 119, // Added in 6.2
```

Comment:
Added in 6.3?

Reply:
Ah, yes. Jakob added 6.2 after I wrote this doc and I missed updating this detail.